### DISCUSSION TITLE
CLIP-MoCo - cool idea

### DISCUSSION THREAD

**Discussion by johnyquest7** [KeremTurgutlu/self_supervised#36] [General]
@KeremTurgutlu CLIP-MoCo is a great idea, planning to try it soon.
You are an inspiration.

**Comment by KeremTurgutlu** [comment]
I didn't got better or worse results with it, but worth trying on different datasets/problems than mine. Let us know how it goes!

**Reply by johnyquest7** [reply]
swav worked best for my dataset. 
Didn't get a chance to try CLIP. Need to reformat the data for that.

**Reply by KeremTurgutlu** [reply]
I believe CLIP would be a better option when you a descriptive relatively longer text as opposed 1-2-gram finite set of class labels.

**Reply by johnyquest7** [reply]
I am in the process of creating a description for all the images based on the tags. If you ever do a full tutorial for CLIP using a custom dataset, please post it here

**Reply by KeremTurgutlu** [reply]
Sure!
